nucliadb/nucliadb/search/api/v1/search.py

```diff
@@ -169,6 +169,12 @@ async def search(
     audit = get_audit()
     timeit = time()
 
+    if len(item.query) > 0 and len(item.query) < 3:
+        raise HTTPException(
+            status_code=422,
+            detail="Query needs to be bigger than 2 or 0",
```

Comment:
that's really a weird message, `Query needs to be bigger than 2 or 0` sounds like bigger than 2 and bigger than 0, maybe `Query needs to be bigger than 2 or empty` would be less confusing